worlds/pokemon_emerald/data.py

```diff
@@ -25,13 +25,20 @@
 }
 """These maps exist but don't show up in the rando or are unused, and so should be discarded"""
 
-POSTGAME_MAPS = {
+OUT_OF_LOGIC_MAPS = {
     "MAP_DESERT_UNDERPASS",
     "MAP_SAFARI_ZONE_NORTHEAST",
     "MAP_SAFARI_ZONE_SOUTHEAST",
     "MAP_METEOR_FALLS_STEVENS_CAVE",
+    "MAP_MIRAGE_TOWER_1F",
+    "MAP_MIRAGE_TOWER_2F",
+    "MAP_MIRAGE_TOWER_3F",
+    "MAP_MIRAGE_TOWER_4F",
 }
-"""These maps have encounters and are locked behind beating the champion. Those encounter slots should be ignored for logical access to a species."""
+"""
+These maps have encounters and are locked behind beating the champion or missable.
```

Comment:
Minor correction
```suggestion
These maps have encounters and are locked behind beating the champion or are missable.
```

Reply:
ftr, I think it was already "correct" if you read as "things are (a or b)" instead of "things are a or things are b"